core/src/templates/components/brand-bar/brand-bar.twig

```diff
@@ -14,6 +14,12 @@
  * - modifier_class: Additional css classes to change look and behaviour.
  */
 #}
+{% if external_link_text is empty -%}
+  {%- set external_link_text -%}
+    <span class="su-brand-bar__link--external">(link is external)</span>
```

Comment:
Strings must be translated.
```suggestion
    <span class="su-brand-bar__link--external">({{ "link is external"|t }})</span>
```

Reply:
Are the round brackets neccessary? "()"

Reply:
According to BEM `--` is used for a modifier to an existing element. This is a whole new element. Perhaps name it `su-brand-bar__link-a11y`?

Reply:
Ignore the "strings must be translated" comment. That is a Drupal thing, and not a Decanter thing.

Reply:
@sherakama ,
With regard to the round brackets (often known as parenthesis), I'm seeing this implemented about half the time. Parenthesis usually indicate that the information within is tangental, and I kind of like that convention. 
If you take this as an example:

I find this harder to read:
<a href="1040.pdf">Tax form 1041 PDF</a>

Than this:
<a href="1040.pdf">Tax form 1041 (PDF)</a>

As we move forward with other icons, I'd like to keep the round brackets "()" as a convention to indicate that the information about the icon is tangental.

Reply:
> round brackets (often known as parenthesis)

Lulz

Reply:
![Screen Shot 2020-08-27 at 1 54 16 PM](https://user-images.githubusercontent.com/550602/91493889-d243eb80-e86c-11ea-97fc-bd2876d34a9e.png)